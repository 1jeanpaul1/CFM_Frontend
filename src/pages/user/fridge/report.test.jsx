import FridgeReportPage from './report';
import { screen, render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

describe('Report', () => {
  it('should render with expected text', () => {
    const { asFragment } = render(<FridgeReportPage a={true} />);

    expect(screen.getByText('Fridge Status Report')).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
