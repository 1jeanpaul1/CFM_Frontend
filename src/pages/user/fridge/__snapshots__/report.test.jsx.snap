// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Report should render with expected text 1`] = `
<DocumentFragment>
  <p>
    Fridge Status Report
  </p>
</DocumentFragment>
`;
